src/status_im/contexts/wallet/collectible/utils.cljs

```diff
@@ -79,3 +79,10 @@
                      acc)))
                {})
        vals))
+
+(defn sort-collectibles-by-name
+  [collectibles]
+  (sort-by (fn [collectible]
+             (let [name (-> collectible :collectible-data :name)]
+               [(if (or (nil? name) (empty? name)) 1 0) name]))
```

Comment:
If name is a string, you might want to use `seq` function.

Reply:
done